nservicebus/messaging/routing.md

```diff
@@ -1,172 +1,68 @@
-
-
-### Endpoint mapping layer
-
-Mapping a message type to the destination endpoint is the logical side of the routing. It tells NServiceBus to which endpoint a given type of command should be sent or, when it comes to events, which endpoint is responsible for publishing it.
-
-
-#### Static routes
-
-The most common way of specifying the type mapping is through static routes. Most of the times the routing should specify the destination endpoint
-
-snippet:Routing-StaticRoutes-Endpoint
-
-When necessary (e.g. for integration with legacy systems) the static routing to a given transport-level address can be configured explicitly. However this should be avoided in most scenarios and the address resolution should be delegated to lower layers of routing.
-
-snippet:Routing-StaticRoutes-Address
-
-
-#### Fallback route
-
-A fallback route can be registered to resolve routes in case no static routes have been found for a given message type. The fallback route provides a convenient extension for NServiceBus add-ons. To add a fallback route, pass a function that takes a `Type` and a `ContextBag` containing the context of current message processing and returns a destination.
-
-snippet:Routing-DynamicRoutes
-
-Following example shows how to implement a shared-store based routing where destinations of messages are managed e.g. in a database.
-
-snippet:Routing-CustomRoutingStore
-
-NOTE: The function passed to the `SetFallbackRoute` call is executed **each time** a message is sent so it is essential to make sure it is performant (e.g. by caching the results if getting the result requires crossing a process boundary).
+## Routing concepts
 
+NServiceBus routing consists of two layers, logical and physical. Logical routing defines which logical endpoint should receive a given outgoing message. Physical routing defines to which actual physical instance of the selected endpoint should the message be delivered. While logical routing is a developer's concern, physical routing is controlled by operations. Apart from MSMQ transport physical routing usually is handled automatically by NServiceBus and does not require any configuration. To read more about physical routing with MSMQ go [here](/nservicebus/msmq/routing.md).
 
-### Endpoint instance mapping layer
 
-The purpose of the endpoint instance mapping layer is to provide information about physical deployments of a given endpoint. Endpoint instance mapping is optional for most transports in most cases because of built-in conventions. An example of situation where it is required is a scaled out endpoint using MSMQ transport. Other endpoints that send messages to it need to be aware of its instances.
+## Command routing
 
+As described [here](/nservicebus/messaging/messages-events-commands.md), NServiceBus distinguishes several types of messages. Command messages are always routed to a single destination. 
```

Comment:
> Command messages are always routed to a single destination.

Single logical endpoint or something along those lines would be more consistent with other articles.